### DISCUSSION TITLE
Error adding certificate to deployment slot

### DISCUSSION THREAD

**Discussion by stefankip** [rcl-ssl/docs#2] [Q&A] [answered]
I have this web app with a staging deployment slot. It has a binding `staging.domain.nl` without an SSL certificate.
So I'm trying to add a certificate via RCL (using the Azure App Services option), enter the staging domain and select the staging deployment slot as Site Name.
In the end the request fails and this is the output:

```
Error:Failed to create app service certificate for host : staging.domain.nl on siteId : 815 with challenge : HTTP for challenge : HTTP. Failure in AzureService in AzureHostnameBindingService : Get. Failed response : NotFound in AzureHostnameBindingService : Get. . Recommendation : none
Information:Waiting for process to complete ****
Connection Id : 654
```

I don't know what to do, the logging from RCL isn't really helpful. Any suggestions?

**Comment by rcladmin** [comment]
This error is being investigated.

**Comment by rcladmin** [comment, marked as answer]
This error has been resolved. You can go ahead a create a SSL/TLS certificate for a App Service in a deployment slot.
